samples/2-ConnectToGraph.ps1

```diff
@@ -6,6 +6,9 @@ Connect-Graph
 # Try to Get-User
 Get-MgUser
 
+# Search for delegated permissions related to sites
+Find-MgGraphPermission sites
```

Comment:
The sample below returns both delegated and application permissions. PermissionType should be specified.

```suggestion
# Search for delegated permissions related to sites
Find-MgGraphPermission sites -PermissionType Delegated
```